dirty_cat/_fast_hash.py

```diff
@@ -47,15 +48,20 @@ def gen_atom(atom_len, seed=0):
     return atom
 
 
-def ngram_min_hash(string, ngram_range=(2, 4), seed=0, return_minmax=False):
+def ngram_min_hash(
+    string: str,
+    ngram_range: Tuple[int, int] = (2, 4),
+    seed: int = 0,
+    return_minmax=False,
+):
     """
     Compute the min/max hash of the ngrams of the string.
 
     Parameters
     ----------
     string : str
         String to encode.
-    ngram_range : tuple (min_n, max_n), default=(2, 4)
+    ngram_range : 2-tuple of int, default=(2, 4)
```

Comment:
To me, given how the [numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) is worded, the information after the column is strictly about the parameter's type.
The actual use of this variable (as min/max bounds) is instead explained (and more explicitly, for example whether the bounds are included or not) in the docstring beneath.
As for the `2-tuple`, it could indeed be replaced by `tuple of (int, int)`. Doesn't matter to me, they mean the same :)